README.md

```diff
@@ -193,42 +248,44 @@ To run the above benchmarks on your machine, see instructions in the [examples](
+    To run a physics example in python (after building with "Physics simulation via Bullet"):
     ```bash
     python examples/example.py --scene /path/to/data/scene_datasets/habitat-test-scenes/skokloster-castle.glb --enable_physics
     ```
-    Note that in this mode the agent will be frozen and oriented toward the spawned physical objects. Additionally, '--save_png' can be used to output agent visual observation frames of the physical scene to the current directory.
+    Note that in this mode the agent will be frozen and oriented toward the spawned physical objects. Additionally, `--save_png` can be used to output agent visual observation frames of the physical scene to the current directory.
 
-We also provide a docker setup for habitat-stack, refer to [habitat-docker-setup](https://github.com/facebookresearch/habitat-api#docker-setup).
 
-## Rendering to GPU Tensors
+## Common testing issues
 
-We support transfering rendering results directly to a [PyTorch](https://pytorch.org/) tensor via CUDA-GL Interop.
-This feature is built by when Habitat-Sim is compiled with CUDA, i.e. built with `--with-cuda`.  To enable it, set the
-`gpu2gpu_transfer` flag of the sensor specification(s) to `True`
+- If you are running on a remote machine and experience display errors when initializing the simulator, e.g.
+   ```bash
+    X11: The DISPLAY environment variable is missing
+    Could not initialize GLFW
+   ```
 
+  ensure you do not have `DISPLAY` defined in your environment (run `unset DISPLAY` to undefine the variable)
 
-This is implemented in a way that is reasonably agnostic to the exact GPU-Tensor library being used, but we currently have only implemented support for PyTorch.
+- If you see libGL errors like: 
 
+   ```bash
+    X11: The DISPLAY environment variable is missing
+    Could not initialize GLFW
+   ```
 
+    chances are your libGL is located at a non-standard location. See e.g. [this issue](https://askubuntu.com/questions/541343/problems-with-libgl-fbconfigs-swrast-through-each-update). 
 
+## Rendering to GPU Tensors
 
-## Developer installation and getting started
+We support transfering rendering results directly to a [PyTorch](https://pytorch.org/) tensor via CUDA-GL Interop.
+This feature is built by when Habitat-Sim is compiled with CUDA, i.e. built with `--with-cuda`.  To enable it, set the
+`gpu2gpu_transfer` flag of the sensor specification(s) to `True`
 
-1. Clone the repo.
-1. Install numpy in your python env of choice (e.g., `pip install numpy` or `conda install numpy`)
-1. Install dependencies in your python env of choice (e.g., `pip install -r requirements.txt`)
-1. If you are using a virtual/conda environment, make sure to use the same environment throughout the rest of the build
-1. Build using `./build.sh` or `./build.sh --headless` for headless systems (i.e. without an attached display) or if you need multi-gpu support
```

Comment:
Using `./build.sh` for development is important part.

Reply:
Seconded.  Due to silliness with c++ libraries and python,  `pytest` doesn't work with the `python setup.py install` build method.

Reply:
Agreed. I wasn't suggesting removing build.sh. I was only suggesting dropping this separation between "Quick Install" and "Developer Install". 

I'll add the information about "build.sh" back in the readme.